Makefile.toml

```diff
@@ -246,7 +246,7 @@ for GLOB_DIR in ${PARAMS}; do
     cd "$ROOT/api_tests/e2e/rfc003/"
     for DIR in ${GLOB_DIR}; do
         cd "$ROOT";
-        ./api_tests/npm_wrapper.sh "./e2e/rfc003/${DIR}"
+        [[ -d "./api_tests/e2e/rfc003/${DIR}" ]] && ./api_tests/npm_wrapper.sh "./e2e/rfc003/${DIR}"
```

Comment:
❓  Does this actually stop if from creating a new directory when the thing doesn't exist?

Reply:
It actually does not proceed if the directory does not exist.
If it does not proceed, no directory will be created.
Feel free to (re)test locally :)